lms/tasks/email_digests.py

```diff
@@ -128,9 +123,7 @@ def send_instructor_email_digests(
     :param h_userids: the h_userid's of the instructors to email
     :param updated_after: the beginning of the time period as an ISO 8601 format string
     :param updated_before: the end of the time period as an ISO 8601 format string
-
-    :param override_to_email: send all the emails to this email address instead
-        of the users' email addresses (this is for test purposes)
+    :param kwargs: other keyword arguments to pass to DigestService.send_instructor_email_digests()
```

Comment:
I changed the Celery task to pass arbitrary `**kwargs` to the service, so that we don't have to update the task every time we update the service's kwargs (the task actually processes the `h_userids`, `updated_after` and `updated_before` kwargs itself so those are separated from the `**kwargs`).